### ISSUE TITLE
"CreateAppHost" task failed unexpectedly

### ISSUE THREAD

**Issue by jtsom** [OmniSharp/omnisharp-vscode#4712]
Issue Type: <b>Bug</b>

Opening default BlazorApp results in an error in the OmniSharp console

Extension version: 1.23.14
VS Code version: Code - Insiders 1.60.0-insider (c08913337cb8b95493be8e99d62e0b3241563f6b, 2021-08-19T05:17:33.223Z)
OS version: Darwin x64 20.6.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz (8 x 2900)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (2.68GB free)|
|Process Argv|--crash-reporter-id d39607bf-1df3-4ebd-aedf-92428e011293|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383:30185418
pythonvspyt602:30291494
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30291487
pythontb:30258533
pythonvspyt551cf:30291413
pythonptprofiler:30281269
vshan820:30294714
pythondataviewer:30285072
pythonvsuse255:30319630
vscod805cf:30301675
pythonvspyt200:30323110
vsccppwtct:30312693
pythonvssor306:30340298
bridge0708:30335490
pygetstartedt2:30353727
dockerwalkthru:30348713
bridge0723:30353136
pythonrunftest32cf:30353182
pythonf5test824:30353183
javagetstartedt:30350119

```

</details>

**Comment by filipw** [comment]
I don't think this is an OmniSharp problem, it looks like https://github.com/dotnet/runtime/issues/55992 which is a change in .NET 6 runtime.

Do you have .NET 6 Preview 7 installed?
Did you try cleaning the obj/bin folders, rebuilding the app from the command line and then opening the project in the C# extension in VS Code again?

**Comment by jtsom** [comment]
Yes, I do (from the Visual Studio Mac Preview. When I added a global.json file to the project, targeting the .NET 5.0.400 SDK, the error went away.
